src/molecules/checkbox/Checkbox.scss

```diff
@@ -43,6 +41,7 @@ $checkbox__icon-transition   : $transition-base !default;
     align-items: center;
     width: $checkbox__item-size;
```

Comment:
sth wronk with variables names. Should be `$checkbox__icon-width` the same for other